### DISCUSSION TITLE
Code formating

### DISCUSSION THREAD

**Discussion by vissarion** [GeomScale/volesti#205] [General]
**Is your feature request related to a problem? Please describe.**
The code contains many styles since we have a few contributors now. So a code formatting tool is needed.

**Describe the solution you'd like**
Clang-format could be used with some well known style, e.g. google format.

**Comment by codeakki** [comment]
may I work on this issue if u guide me a little bit because I'm new here .

**Comment by vissarion** [comment]
@codeakki you are welcome to propose a PR that changes the format of a small part of library e.g. tests. Please read our contributions guide.
